### PR TITLE
[AUT-3124] extend charts with fission deployment

### PR DESCRIPTION
**Pull request by ppodbielski-cloudentity**

### CODE REVIEW

**Review by mbilski — CHANGES_REQUESTED** [review]
The values.yaml should contain only the minimal configuration and allow to configure critical points.

**Comment by ppodbielski-cloudentity** [comment]
`# Source: acp/templates/fission.yaml
apiVersion: fission.io/v1
kind: Environment
metadata:
  name: nodejs
spec:
  version: 1
  imagepullsecret: 
  poolsize: 3
  terminationGracePeriod: 1
  runtime:
    image: docker pull fission/node-env:latest
    podspec:
      containers:
        - name: nodejs
          imagePullPolicy: Always

apiVersion: fission.io/v1
kind: Environment
metadata:
  name: rego
spec:
  version: 1
  imagepullsecret: artifactory
  poolsize: 3
  terminationGracePeriod: 1
  runtime:
    image: docker.cloudentity.io/rego-env:latest3
    podspec:
      containers:
        - name: rego
          imagePullPolicy: Always`

**Review by artursmolarek — APPROVED** [review]

**Review by jkotiuk — APPROVED** [review]

**Review by mbilski — APPROVED** [review]

**Review by mbilski — CHANGES_REQUESTED** [review]
I don't see any changes in acp deployment which would enable fission.
What about RBAC roles that acp needs?

**Review by mbilski — APPROVED** [review]

**Comment by mbilski** [comment]
FYI this configuration works only with acp 1.16+, not 1.15.x

**Review by jkotiuk — APPROVED** [review]

File: charts/acp/README.md [hunk 1]
```diff
@@ -124,3 +124,53 @@ acp:
...
+>
+>```yaml
+>  ## Define namespace where the Fission Environment can be deployed
+>  ##
+>  namespace: acp-faas
+>```
+
+### Configuration
+
+To see all configurable options with detailed comments, visit the chart's [values.yaml](./values.yaml), fission sectoion. By default Fission Environment for ACP is disabled. To enable switch the flag `fission.enabled` to `true`:
```
Comment (plandzwojczak-cloudentity) [inline charts/acp/README.md:167]: typo

File: charts/acp/README.md [hunk 2]
```diff
@@ -124,3 +124,53 @@ acp:
...
+>  ## Define namespace where the Fission Environment can be deployed
+>  ##
+>  namespace: acp-faas
+>```
+
+### Docker Pull Credentials
+
+Fission environment defines Pod that uses a Secret to pull an image from a private Docker registry or repository, please refer to the documentation for that mechanism ([Pull an Image from a Private Registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/))
+
+To manually configure Docker credentials, first createreate a Secret by providing credentials on the command line:
```
Comment (plandzwojczak-cloudentity) [inline charts/acp/README.md:151]: typo `createreate`

**Review by plandzwojczak-cloudentity — APPROVED** [review]

**Pull request merged**